README.md

```diff
@@ -2,60 +2,106 @@
-| [`@turnkey/api-key-stamper`](/packages/api-key-stamper)                           | [![npm](https://img.shields.io/npm/v/@turnkey/api-key-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/api-key-stamper)                           | Provide API key signatures over Turnkey requests                        | [CHANGELOG](/packages/api-key-stamper/CHANGELOG.md)              | Turnkey primitives |
-| [`@turnkey/iframe-stamper`](/packages/iframe-stamper)                             | [![npm](https://img.shields.io/npm/v/@turnkey/iframe-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/iframe-stamper)                             | Provide API key signatures over Turnkey requests within iframe contexts | [CHANGELOG](/packages/iframe-stamper/CHANGELOG.md)               | Turnkey primitives |
-| [`@turnkey/webauthn-stamper`](/packages/webauthn-stamper)                         | [![npm](https://img.shields.io/npm/v/@turnkey/webauthn-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/webauthn-stamper)                         | Provide Webauthn signatures over Turnkey requests                       | [CHANGELOG](/packages/webauthn-stamper/CHANGELOG.md)             | Turnkey primitives |
-| [`@turnkey/wallet-stamper`](/packages/wallet-stamper)                             | [![npm](https://img.shields.io/npm/v/@turnkey/wallet-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/wallet-stamper)                             | Provide wallet signatures over Turnkey requests                         | [CHANGELOG](/packages/wallet-stamper/CHANGELOG.md)               | Turnkey primitives |
+## Overview
+
+The Turnkey SDK includes functions and clients with methods that construct, authenticate and execute requests to the Turnkey API with support for different ecosystems and methods of authentication. It consists of the [Primary Turnkey Web SDK Packages](#primary-turnkey-web-sdk-packages) which expose the main functionality required to build Turnkey-powered applications in different web environments, the [Chain/Ecosystem-Specific Signing Packages](#chainecosystem-specific-signing-sdk-packages) which expose signers with support for specific ecosystems, built on top of our Web SDK, and the [Advanced Functionality SDK Packages](#advanced-functionality-sdk-packages) which exposes lower level functionality that is leveraged by our Primary Web SDK Packages for those with highly-specific implementations looking to use them.
+
+The below diagram helps visualize the packages in our SDK organized by the functionality they expose.
+
+<img src="./img/sdk-map.png" alt="homepage screenshot" width="1000px">
```

Comment:
I like this diagram a lot but I'd change a couple of things:
* `@turnkey/react-native-passkey-stamper` isn't a "main" package. It's just a stamper so it should be moved next to `@turnkey/webauthn-stamper` and `@turnkey/api-key-stamper`
* The arrows need labeling. I think the vertical arrow should go down and read "uses" or "imports". And the horizontal arrow can be bidirectional and read "integrates with"? Ecosystem packages generally consumes our other packages, but sometimes it's the other way around

There's also a marketing opportunity here with ecosystem/blockchain logos next to each ecosystem package (def nice-to-have!)

Reply:
+1; I think the "Advanced Functionality" label might imply a higher order, when in reality the hierarchy is such that the stampers are prereqs for all the abstractions built atop them (conveying this hierarchy via arrows/etc is huge so thanks for approaching this in such a way). Stampers + `@turnkey/http` could be considered Primitives/foundational packages

Technically, `@turnkey/http` isn't quite a stamper but rather a library containing an http client that allows for (easier) interaction with Turnkey endpoints. And I think the utilities SDK packages could be in a separate bucket of their own.

Reply:
@r-n-o On the logos -- i put some of them up but it looks a bit lopsided because some of these are actually related to a chain ecosystem with a logo like solana or ethers, but viem for example or eip stamper etc would either also have ethereum logos or none at all. For now I think it would be best to move forward without the logos! Lmk if you have strong opinions otherwise.

Reply:
I'm down to move forward without. Makes sense 👍